<launch>
  <arg name="markerSize"      default="0.05"/>
  
  <include file="$(find aruco_ros)/launch/marker_publisher.launch.launch">
        <arg name="markerSize" value="$(arg markerSize)"/>
  </include>
  
  <node pkg="erc_remote_maintenance" type="test_drive2.py"  name="robot_controller" output="screen"/>

</launch>
